apps/ex_wire/test/ex_wire/packet/get_block_bodies_test.exs

```diff
@@ -1,4 +1,4 @@
 defmodule ExWire.Packet.GetBlockBodiesTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
```

Comment:
if you move the tests here, where they belong, they can run async.

Reply:
Moving tests into the test from doctest wouldn't make them be able to run async. The change would be on not using named processes that conflict with one-another, right?

Reply:
Ah yes, didn’t notice it’s named!